themes/root/css/icon-helper.css

```diff
@@ -6,6 +6,7 @@
 .icon--font {
     display: inline-block;
     width: 1.25em;
+    line-height: inherit;
```

Comment:
Since this entire file has been removed from #1962, does this line need to be translated somewhere over there? Just mentioning it so we don't lose track of it.